resources/oracle-to-psql-converters/final_output-20240305.txt

```diff
@@ -0,0 +1,129 @@
+INSERT INTO "organisations" ("sbi","addressLine1", "addressLine2", "addressLine3", "city", "county", "postcode", "emailAddress", "frn", "name", "updated")
+VALUES
+ (106613073,'COOGIE FARM','CHURCH END','CAWOOD','SELBY',null,'Z03 3PN','email1@yahoo.co.uk',1102358109,'FARMNAME1',to_date('14-FEB-24 01:34:17','DD-MON-YY HH:MI:SS')),
+ (106268747,'OETER COTTAGE',null,'HAMSTERLEY','BISHOP AUCKLAND',null,'CC13 3KA','email12@graysllp.co.uk',1101007842,'FARMNAME12',to_date('14-FEB-24 01:34:17','DD-MON-YY HH:MI:SS')),
+ (107069599,'LAMHAM FARM','COLD ASHBY ROAD','WEST HADDON','NORTHAMPTON',null,'GG6 7RE','email13@gmail.com',1100252193,'FARMNAME13',to_date('14-FEB-24 01:34:17','DD-MON-YY HH:MI:SS')),
+ (200211430,'BOBRIDGE',null,'COWAN BRIDGE','CARNFORTH',null,'FQ6 2SS','email14@gmail.com',1104376954,'FARMNAME14',to_date('14-FEB-24 01:34:17','DD-MON-YY HH:MI:SS')),

```

Comment:
same as before, we can cross reference, we can search for some of those details and come across data we don't want on display, please think about replacing anything that could be identifiable with absolute dummy data